common/djangoapps/student/helpers.py

```diff
@@ -230,3 +231,11 @@ def get_next_url_for_login_page(request):
         # be saved in the session as part of the pipeline state. That URL will take priority
         # over this one.
     return redirect_to
+
+
+def invalidate_access_token(user):
```

Comment:
You're not invalidating them. You're deleting them. Rename this to something along the lines of `destroy_oauth_tokens`.